### PR TITLE
Orders test

### PR DESCRIPTION
**Pull request by dieie32**

### CODE REVIEW

File: src/redux/orders/test/orders-reducer.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import {
+  setOrder,
+  setOrderList,
+  setOrderLoading,
+  setOrderError
+} from '../orders.actions';
+
+describe('orders reducer tests', () => {
+  it('should return default state', () => {
+    expect(ordersReducer()).toEqual(initialState);
```
Comment (moran711) [inline src/redux/orders/test/orders-reducer.test.js:11]: Please pass your `initialState` variable to reducer.

File: src/redux/orders/test/orders.variables.js [hunk 2]
```diff
@@ -0,0 +1,218 @@
+export const operationsAllOrdersLimit = 5;
+export const operationsAllOrdersSkip = 0;
+export const operationsAllOrdersFilter = {
+  orderStatus: ['DELIVERED']
+};
+
+export const expectedResultOperationsAllOrders = {
```
Comment (moran711) [inline src/redux/orders/test/orders.variables.js:7]: Please rename this variable to `orders` or similar.

File: src/redux/orders/test/orders.variables.js [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  data: {
+    getAllOrders: {
+      ...setOrderListFakeData
+    }
+  }
+};
+
+export const getOrderListFakeData = {
```
Comment (moran711) [inline src/redux/orders/test/orders.variables.js:15]: Please rename this variable to `orderList` or similar.

File: src/redux/orders/test/orders.variables.js [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+  limit: 5,
+  skip: 0,
+  filter: {
+    orderStatus: 'DELIVERED'
+  }
+};
+
+export const getOrderListFakeDataSaga = {
```
Comment (moran711) [inline src/redux/orders/test/orders.variables.js:23]: Do you need this variable, if you have `getOrderListFakeData`?

File: src/redux/orders/test/orders-reducer.test.js (hunk 1 above)
Comment (moran711) [inline src/redux/orders/test/orders-reducer.test.js:11]: Please pass your `initialState` to `ordersReduser` as an argument.

File: src/redux/orders/test/orders.variables.js [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+  limit: 5,
+  skip: 0,
+  filter: {
+    orderStatus: 'DELIVERED'
+  }
+};
+
+export const setOrderListFakeData = {
```
Comment (moran711) [inline src/redux/orders/test/orders.variables.js:31]: Please rename this variable.

File: src/redux/orders/test/orders.variables.js [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+
+export const fakeOrders = {
+  data: {
+    getAllOrders: {
+      ...setOrderListFakeData
+    }
+  }
+};
+
+export const getOrdersFakeError = {
```
Comment (moran711) [inline src/redux/orders/test/orders.variables.js:120]: Please rename this variable.

**Review by DmytroDidukh — APPROVED** [review]

File: src/redux/orders/orders.sagas.js [hunk 7]
```diff
@@ -55,17 +40,24 @@ export function* handleOrdersListLoad({ payload }) {
       payload.limit,
       payload.filter.orderStatus
     );
-    if (orders.errors) {
```
Comment (moran711) [inline src/redux/orders/orders.sagas.js:58]: Why you remove this from saga?

**Review by DmytroDidukh — APPROVED** [review]

File: src/redux/orders/orders.sagas.js (hunk 7 above)
Comment (dieie32) [reply src/redux/orders/orders.sagas.js:58]: it doesn't work

**Review by dimaPalii — APPROVED** [review]

File: src/redux/orders/orders.sagas.js (hunk 7 above)
Comment (moran711) [reply src/redux/orders/orders.sagas.js:58]: It should work if we have errors array in response.

**Review by moran711 — APPROVED** [review]

**Pull request merged**